src/sparseml/transformers/masked_language_modeling.py

```diff
@@ -688,13 +694,17 @@ def compute_metrics(eval_preds):
 
     # Training
     if training_args.do_train:
-        checkpoint = None
+        checkpoint = checkpoint_recipe = None
```

Comment:
no need to find the checkpoint recipe in the scripts, these are saved to a standard path that the trainer can find (see `RECIPE_REGEX` and `RECIPE_NAME`)

Reply:
Thanks. This is correct. Resolved.